.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -98,6 +108,24 @@ jobs:
           TARGET_BRANCH: ${{ inputs.target-branch }}
           RELEASE: ${{ inputs.stage == 'release' && 'yes' || 'no' }}
 
+      - uses: oras-project/setup-oras@v1
+        if: (matrix.asset == 'agent') && (inputs.push-to-registry == 'yes')
```

Comment:
Can we pin the oras-version - I kinda feel like it should be tracked centrally in versions.yaml's `externals` section to sync it with https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/local-build/dockerbuild/install_oras.sh#L18 as well if that's ok?

Reply:
do you mean pinning it to v1.2 here is sufficient? or should there be a `external.oras` entry in versions yaml that's used everywhere?

Reply:
I think there should be an `external.oras` entry in `version.yaml`, but if that needs to be a separate PR then that's ok

Reply:
I took a look, it doesn't seem like we have versions.yaml processing in this particular corner of ci machinery. it would be trivial to do that on the workflow, but the propagation of such a value gets hairy for `install_oras.sh` since this is invoked via layers of Makefiles and Dockerfiles, and while the script claims to use versions.yaml, it's using a hardcoded version afaict.

So I'd suggest fixing it to version 1.2.0 here, too. also, for s390x things are special. I will also remove the setup-oras step for s390x, as we did on guest-components. the runner should ship 1.2.0 already.

Reply:
@BbolroC - can you confirm whether oras is installed in the kata self-hosted s390x runner please?

Reply:
Yes, it is installed for all s390x runners. Thanks for asking.